### ISSUE TITLE
More terminals with sixel support; more sixel implementations

### ISSUE THREAD

**Issue by ghost** [libsixel/libsixel#58]
Not sure where these fit, as there is a lot of overlap in "Supported terminals", "Other implementations", and "Projects using SIXEL".

More terminals:
* SyncTERM (since version 1.1) - http://www.syncterm.net/
* xterm.js when using xterm-image-addon - https://github.com/jerch/xterm-addon-image
* foot - https://codeberg.org/dnkl/foot
* contour - https://github.com/contour-terminal/contour

More encoder implementations:
* [rasterm](https://github.com/BourgeoisBear/rasterm) - Go

**Comment by j4james** [comment]
Some more terminals with sixel support:

* Alacritty (via the ayosec graphics fork) - https://github.com/ayosec/alacritty/tree/graphics
* IBM Personal Communications - https://www.ibm.com/support/pages/ibm-personal-communications
* Rxvt (via the saitoha sixel fork) - https://github.com/saitoha/rxvt-unicode-sixel
* VTE (built with the sixel option) - https://gitlab.gnome.org/GNOME/vte/

Also, if you're going to be updating that page, note that WRQ Reflection has long since changed ownership and is now known as Microfocus Reflection Desktop (https://www.microfocus.com/en-us/products/reflection-desktop/). There are actually a bunch of Reflection products for different platforms, but I don't think they all support sixel - I know Reflection Desktop definitely does.

**Comment by klamonte** [comment]
Also update/add RLogin's github repo link: https://github.com/kmiya-culti/RLogin

(Thank you @j4james !)

**Comment by christianparpart** [comment]
Hm.. having some thoughts on this:

- xterm.js: isn't the addon kinda orphaned. Not maintained and no intend to do so, sadly? (I might be wrong)
- Alacritty's PR: If i read some comments in that PR (and other sites) correctly, that's sadly unlikely to be merged. Or is it?
- VTE: did any TE pick this mode up already, e.g. gnome-terminal?

**Comment by j4james** [comment]
The last comments I've seen on the issue tracker for the xterm.js image addon were from February, which isn't that long ago, and it definitely didn't sound like it was abandoned to me.

I suspect the Alacritty sixel PR is not very likely to be merged in the main repo, but there are still people actively maintaining forks with the sixel support included.

As for VTE, I think it could theoretically be used in any VTE-based terminal as long as you're willing to build it yourself. I've only tested in the VTE demo app though, so I might be wrong about that.

**Comment by Kreijstal** [comment]
> Some more terminals with sixel support:
> 
>     * Alacritty (via the ayosec graphics fork) - https://github.com/ayosec/alacritty/tree/graphics
> 
>     * IBM Personal Communications - https://www.ibm.com/support/pages/ibm-personal-communications
> 
>     * Rxvt (via the saitoha sixel fork) - https://github.com/saitoha/rxvt-unicode-sixel
> 
>     * VTE (built with the sixel option) - https://gitlab.gnome.org/GNOME/vte/
> 
> 
> Also, if you're going to be updating that page, note that WRQ Reflection has long since changed ownership and is now known as Microfocus Reflection Desktop (https://www.microfocus.com/en-us/products/reflection-desktop/). There are actually a bunch of Reflection products for different platforms, but I don't think they all support sixel - I know Reflection Desktop definitely does.

I think VTE took down the sixel support?

**Comment by j4james** [comment]
> I think VTE took down the sixel support?

Not exactly. The Sixel code is all still there. It just stopped working when they added native GTK4 drawing (see [issue 2717](https://gitlab.gnome.org/GNOME/vte/-/issues/2717)). And it wasn't considered a priority to fix, because it was never officially released in the first place - it's always been considered a work in progress.

That said, I don't think there's anyone working on it anymore, so I can't imagine we're going to get Sixel back in VTE anytime soon. But if you're desperate, you can still build it from an older branch.
